### PR TITLE
API: Delegation | update delegation

### PR DESCRIPTION
**Pull request by Sotatek-GiangNguyen7**

### CODE REVIEW

**Comment by Sotatek-GiangNguyen7** [comment]
image

File: src/main/java/microservices/addresses/steps/AddressSteps.java [hunk 1]
```diff
@@ -17,6 +17,7 @@ public class AddressSteps extends BaseSteps {
     @Step("get data for address")
     public AddressSteps getDataForAddress(Object address){
         sendGet(Endpoints.AddressesApi.ADDRESS_URI, AddressConstants.ADDRESS, address);
+        System.out.println(getResponse().getBody().print());
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/addresses/steps/AddressSteps.java]: remove it

File: src/main/java/microservices/addresses/steps/AddressTransactionSteps.java [hunk 2]
```diff
@@ -16,11 +18,14 @@
...
 
 import static constants.DateFormats.DATE_FORMAT;
+import com.google.gson.JsonElement;
+
 
 public class AddressTransactionSteps extends BaseSteps {
     @Step("get the transaction of address")
     public AddressTransactionSteps getTheTransactionOfAddress(Object address){
         sendGet(Endpoints.AddressesApi.ADDRESS_TRANSACTION_URI, AddressConstants.ADDRESS, address);
+        System.out.println(getResponse().getBody().print());
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/addresses/steps/AddressTransactionSteps.java]: remove it

File: src/main/java/microservices/addresses/steps/AddressTransactionSteps.java [hunk 3]
```diff
@@ -73,6 +78,10 @@ public AddressTransactionSteps verifyValueOfAttributeIsCorrectly(AddressTransact
     }
     @Step("verify attribute exist")
     public AddressTransactionSteps verifyAttributeExist(){
+        System.out.println(getResponse().getBody().asString());
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/addresses/steps/AddressTransactionSteps.java]: remove it

File: src/main/java/microservices/addresses/steps/AddressTransactionSteps.java [hunk 4]
```diff
@@ -73,6 +78,10 @@ public AddressTransactionSteps verifyValueOfAttributeIsCorrectly(AddressTransact
...
+        JsonParser parser = new JsonParser();
+        JsonElement jsonObject = parser.parse(getResponse().getBody().asString()).getAsJsonObject();
+        //System.out.println(isKeyPresent(getResponse().getBody().asString(), "hash"));
```
Comment (Sotatek-DavidNguyen4) [inline src/main/java/microservices/addresses/steps/AddressTransactionSteps.java]: remove it

**Review by Sotatek-DavidNguyen4 — APPROVED** [review]

**Pull request merged**